sql/sql_class.h

```diff
@@ -4854,6 +4854,7 @@ class THD: public THD_count, /* this must be first */
     table updates from being replicated to other nodes via galera replication.
   */
   bool                      wsrep_ignore_table;
+  bool                      wsrep_killed;
```

Comment:
A comment that describes the purpose of the flag and the locking rules is missing.

Reply:
It seems that we could protect it with LOCK_thd_data mutex